### ISSUE TITLE
Bug in gcode.c causes Jog issue (using Universal Gcode Sender).

### ISSUE THREAD

**Issue by mattknudstrup** [fra589/grbl-Mega-5X#116]
Line 1193, parenthesis misplaced:
    if (command_dwords & ~(bit(MODAL_GROUP_G3) | bit(MODAL_GROUP_G6 | bit(MODAL_GROUP_G0))) ) { FAIL(STATUS_INVALID_JOG_COMMAND) };
...should be...
    if (command_dwords & ~(bit(MODAL_GROUP_G3) | bit(MODAL_GROUP_G6) | bit(MODAL_GROUP_G0)) ) { FAIL(STATUS_INVALID_JOG_COMMAND) };

**Comment by fra589** [comment]
Hi @mattknudstrup,  
Oops! you must be right. It's a typo, I will correct it ASAP...
Thanks for reporting,
@++;
Gauthier.

**Comment by fra589** [comment]
Hi @mattknudstrup,  
I just pushed a commit correcting this issue, it's was too easy since you reported the bug location :smile:.
Thanks to test and report if OK.
@++;
Gauthier.

**Comment by mattknudstrup** [comment]
Hello Gauthier,
Unfortunately I am using a different fork for the Mega (that I can't even seem to find the source for anymore) that I have upgraded to 8 axis (currently using them all believe it or not).  I found this bug in the source that I'm using and in my debugging found the same error in your code which was actually in the main code base at one time, but was later fixed.  There was another bug in my source but I checked and yours didn't have that one.  Your code should be good to go.  If you really want I could download and check it on a spare Mega.
